### ISSUE TITLE
README Instructions dont work

### ISSUE THREAD

**Issue by devinschumacher** [MarsX-dev/devhunt#136]
the sql dump area is outdated. but even when correcting it, it just errors

**Comment by devinschumacher** [comment]
supabase doesnt allow alot of the permissions that are the thing failing during it:

![Screenshot 2024-04-28 at 02 11 22](https://github.com/MarsX-dev/devhunt/assets/45643901/c036e1d0-d4df-478b-97c6-ec00b84acd60)

**Comment by devinschumacher** [comment]
<img width="913" alt="image" src="https://github.com/MarsX-dev/devhunt/assets/45643901/45675d3f-ad94-40bb-8167-dc922bea7b31">

**Comment by das-mechaniker** [comment]
Yeah, something is broken with this process. I have tried just about everything. Are there any others who were able resolve this? Not just auth issues but also duplicate primary key errors.
